### PR TITLE
Identify SPDX license identifiers for licenses

### PR DESCRIPTION
**Pull request by cajun-rat**

### CODE REVIEW

File: thirdparty.spdx [hunk 1]
```diff
@@ -25,8 +25,8 @@ SPDXID: SPDXRef-googletest
 PackageVersion: 1.8.0
 PackageDownloadLocation: https://github.com/google/googletest/archive/release-1.8.0.zip
 PackageHomePage: https://github.com/google/googletest
-PackageLicenseConcluded: LicenseRef-GoogleLicense
-PackageLicenseDeclared: LicenseRef-GoogleLicense
+PackageLicenseConcluded: BSD-3-Clause
+PackageLicenseDeclared: BSD-3-Clause
```
Comment (patriotyk) [inline thirdparty.spdx]: Good

File: thirdparty.spdx [hunk 2]
```diff
@@ -37,8 +37,8 @@ SPDXID: SPDXRef-jsoncpp
 PackageVersion: 0.6.0
 PackageDownloadLocation: https://github.com/open-source-parsers/jsoncpp/archive/1.6.0.zip
 PackageHomePage: https://github.com/open-source-parsers/jsoncpp
-PackageLicenseConcluded: LicenseRef-jsoncpp
-PackageLicenseDeclared: LicenseRef-jsoncpp
```
Comment (patriotyk) [inline thirdparty.spdx]: jsoncpp license has following prefix:

The JsonCpp library's source code, including accompanying documentation,
tests and demonstration applications, are licensed under the following
conditions...

The author (Baptiste Lepilleur) explicitly disclaims copyright in all
jurisdictions which recognize such a disclaimer. In such jurisdictions,
this software is released into the Public Domain.

In jurisdictions which do not recognize Public Domain property (e.g. Germany as of
2010), this software is Copyright (c) 2007-2010 by Baptiste Lepilleur, and is
released under the terms of the MIT License (see below).

In jurisdictions which recognize Public Domain property, the user of this
software may choose to accept it either as 1) Public Domain, 2) under the
conditions of the MIT License (see below), or 3) under the terms of dual
Public Domain/MIT License conditions described here, as they choose.

File: thirdparty.spdx [hunk 3]
```diff
@@ -48,8 +48,8 @@ SPDXID: SPDXRef-picojson
 PackageVersion: 1.3.0
 PackageDownloadLocation: https://github.com/kazuho/picojson/archive/v1.3.0.zip
 PackageHomePage: https://github.com/kazuho/picojson
-PackageLicenseConcluded: LicenseRef-picojson
-PackageLicenseDeclared: LicenseRef-picojson
```
Comment (patriotyk) [inline thirdparty.spdx]: Good

File: thirdparty.spdx [hunk 4]
```diff
@@ -174,8 +174,8 @@ PackageName: libcurl
 SPDXID: SPDXRef-libcurl
 PackageDownloadLocation: https://github.com/curl/curl/archive/master.zip
 PackageHomePage: https://curl.haxx.se
-PackageLicenseConcluded: LicenseRef-curl
-PackageLicenseDeclared: LicenseRef-curl
+PackageLicenseConcluded: MIT
+PackageLicenseDeclared: MIT
```
Comment (patriotyk) [inline thirdparty.spdx]: It is ISC license with exceptions from MIT license

File: thirdparty.spdx [hunk 5]
```diff
@@ -198,8 +198,8 @@ PackageName: libarchive
 SPDXID: SPDXRef-libarchive
 PackageDownloadLocation: https://github.com/libarchive/libarchive/zipball/master
 PackageHomePage: https://www.libarchive.org/
-PackageLicenseConcluded: LicenseRef-libarchive
-PackageLicenseDeclared: LicenseRef-libarchive
```
Comment (patriotyk) [inline thirdparty.spdx]: It has additional text:

The libarchive distribution as a whole is Copyright by Tim Kientzle
and is subject to the copyright notice reproduced at the bottom of
this file.

Each individual file in this distribution should have a clear
copyright/licensing statement at the beginning of the file.  If any do
not, please let me know and I will rectify it.  The following is
intended to summarize the copyright status of the individual files;
the actual statements in the files are controlling.

* Except as listed below, all C sources (including .c and .h files)
  and documentation files are subject to the copyright notice reproduced
  at the bottom of this file.

* The following source files are also subject in whole or in part to
  a 3-clause UC Regents copyright; please read the individual source
  files for details:
   libarchive/archive_entry.c
   libarchive/archive_read_support_filter_compress.c
   libarchive/archive_write_add_filter_compress.c
   libarchive/mtree.5

* The following source files are in the public domain:
   libarchive/archive_getdate.c

* The build files---including Makefiles, configure scripts,
  and auxiliary scripts used as part of the compile process---have
  widely varying licensing terms.  Please check individual files before
  distributing them to see if those restrictions apply to you.

I intend for all new source code to use the license below and hope over
time to replace code with other licenses with new implementations that
do use the license below.  The varying licensing of the build scripts
seems to be an unavoidable mess.

**Comment by cajun-rat** [comment]
Closing stale PR

**Pull request closed without merging**